### PR TITLE
Initial SpringBoot application set up

### PR DESCRIPTION
**Pull request by hwilliams1**
Initial Spring Boot Application set up for chd-order-api

GCI-768

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,5 +19,21 @@
...
+*.iws
+*.iml
+*.ipr
+
+target/
+.classpath
+.project
+.settings/
+.vscode/
+.factorypath
```
Comment (YiannisPapastergiou) [inline .gitignore:39]: When creating other Spring boot apps, I was told to follow this gitignore file example:

`#Build artifacts
target/
*.jar
*.zip

# Eclipse files
.classpath
.project
.settings
bin/

# IntelliJ files
.idea/
*.iml

# Mac files
.DS_Store
._*
`

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# Start script for chd-order-api
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/chd-order-api/env
+
+# FIXME replace with correct variable name
```
Comment (YiannisPapastergiou) [inline start.sh:12]: the comments in this line and line 31 should be removed?

**Review by YiannisPapastergiou — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+server:
+  servlet:
+    context-path: /chd-order-api
```
Comment (VaughanJackson123) [inline src/main/resources/application.yaml:3]: Any particular reason why we are setting this?

File: src/main/java/uk/gov/companieshouse/chd/order/api/logging/LoggingUtils.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.chd.order.api.logging;
+
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+public class LoggingUtils {
+    
+    public static final String APPLICATION_NAMESPACE="chd-order-api";
+
```
Comment (VaughanJackson123) [inline src/main/java/uk/gov/companieshouse/chd/order/api/logging/LoggingUtils.java:9]: This constant could become `private` given the `getLogger()` method? Not terribly important.

Comment (hwilliams1) [reply src/main/java/uk/gov/companieshouse/chd/order/api/logging/LoggingUtils.java:9]: true

File: start.sh (hunk 2 above)
Comment (hwilliams1) [reply start.sh:12]: true, will do that

File: src/main/resources/application.yaml (hunk 3 above)
Comment (hwilliams1) [reply src/main/resources/application.yaml:3]: It's me being on autopilot when going through the setup...

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**